src/Core/Translation/Storage/Provider/ModuleCatalogueLayersProvider.php

```diff
@@ -303,7 +303,7 @@ private function buildTranslationCatalogueFromLegacyFiles(string $locale): Messa
 
         foreach ($catalogueFromPhpAndSmartyFiles->all() as $currentDomain => $items) {
             foreach (array_keys($items) as $translationKey) {
-                $legacyKey = md5($translationKey);
+                $legacyKey = md5((string)$translationKey);
```

Comment:
```suggestion
                $legacyKey = md5((string) $translationKey);
```
This should fix the csfixer issue

Reply:
Done @matthieu-rolland